test/features/step_definitions/progression_steps.js

```diff
@@ -0,0 +1,9 @@
+const progression = require('../../../lib/progression');
+
+module.exports = function () {
+  this.Then(/^it should be be at the beginning of the job application progression$/, function () {
+    // TODO: this is very clunky, but I want to discuss approach before implementing anything
+    // better
+    return this.browser.assert.text('.progression-status', progression[0]);
```

Comment:
what about something like this `expect(dashboardPage.statusForJob(this.scenarioData.addedJob)).to.equal(progression[0]`